tests/integration/submission/test_upload.py

```diff
@@ -262,6 +266,77 @@ def test_data_file_update_urls(
     assert set(record1.urls) == set(record.urls) or set(new_urls)
 
 
+@pytest.mark.config_toggle(
+    parameters={
+        'CREATE_REPLACEABLE': True
+    }
+)
+def test_data_file_update_recreate_index(
+        client_toggled, pg_driver, cgci_blgsp, indexd_client, submitter):
+    resp_json, sur_entity = data_file_creation(
+        client_toggled, submitter,
+        sur_filename='submitted_unaligned_reads.json')
+
+    assert resp_json['code'] == 201
+
+    node_id = sur_entity['id']
+    old_doc = indexd_client.get(node_id)
+
+    new_version = read_json_data(
+        os.path.join(DATA_DIR, 'submitted_unaligned_reads_new.json')
+    )
+
+    resp = client_toggled.put(
+        BLGSP_PATH, headers=submitter, data=json.dumps(new_version))
+
+    assert resp.status_code == 200
+
+    new_doc_json = indexd_client.get(node_id).to_json()
+    assert old_doc.to_json() != new_doc_json
+    assert old_doc.to_json()['rev'] != new_doc_json['rev']
+    assert new_doc_json['file_name'] == new_version['file_name']
+    assert new_doc_json['hashes']['md5'] == new_version['md5sum']
+    assert new_doc_json['size'] == new_version['file_size']
+
+
+def test_is_updatable_file(client, pg_driver, indexd_client):
+    """Test _is_updatable_file_node method
+    """
+
+    did = 'bef870b0-1d2a-4873-b0db-14994b2f89bd'
+    url = '/some/url'
+
+    # Create dummy file node and corresponding indexd record
+    node = SubmittedAlignedReads(did)
+    indexd_client.create(
+        did=did,
+        urls=[url],
+        hashes={'md5': '0'*32},
+        size=1,
+    )
+    ALLOWED_STATES = [
```

Comment:
I'm assuming this is defined in production code somewhere else. It might be better to import it from there so this doesn't produce false positives in the future.

Reply:
Yeah, I am moving it to sheepdog.globals